src/javascript/detectors/cross_site_scripting/cross_site_scripting.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
should we include `require('node-esapi')` here to make example self contained? Might have to change `const` to `var`